jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -94,13 +94,14 @@ def test_parse_numpy_array_3d(self, imviz_helper, manual_loop):
         n_slices = 5
         slice_shape = (2, 3)
         arr = np.stack([np.zeros(slice_shape) + i for i in range(n_slices)])
+        data_label = 'my_slices'
 
         if not manual_loop:
             # We use higher level load_data() here to make sure linking does not crash.
-            imviz_helper.load_data(arr, data_label='my_slices')
+            imviz_helper.load_data(arr, data_label=data_label)
         else:
             for i in range(n_slices):
-                imviz_helper.load_data(arr[i, :, :], data_label=f'my_slices_{i}', do_link=False)
+                imviz_helper.load_data(arr[i, :, :], data_label='my_slices', do_link=False)
```

Comment:
Should `"my_slices"` here be `data_label`?

Reply:
What do you mean, @bmorris3 ?